.mergify.yml

```diff
@@ -101,6 +101,17 @@ pull_request_rules:
         - files~=^\.mergify\.yml$
     actions:
       delete_head_branch:
+  - name: notify the backport has not been merged yet
+    conditions:
+      - -merged
+      - -closed
+      - author=mergify[bot]
+      - "#check-success>0"
+      - schedule=Mon-Mon 06:00-10:00[Europe/Paris]
+    actions:
+      comment:
+        message: |
+          This pull request has not been merged yet. Could you please review and merge it @{{ assignee | join(', @') }}? 🙏
```

Comment:
Some backports don't have an assignee (for example https://github.com/elastic/beats/pull/28783), should we have a different action for these ones? (Not sure which one)

Reply:
We could create a new one to add a team by default if no assignee?

Reply:
Ok, but I don't know who should be pinged on this action. We can leave this for a follow up.

The only thing to consider here is that assignee may be empty, do we still want to add a comment in this case? It will generate a message like:

> This pull request has not been merged yet. Could you please review and merge it @? 🙏

Reply:
Not reported anymore to those PRs without an assignee. We will work on this in the future with another rule to notify to a more broadly team, maybe...